ton_client/src/net/endpoint.rs

```diff
@@ -46,32 +45,24 @@ impl Clone for Endpoint {
             next_latency_detection_time: AtomicU64::new(
                 self.next_latency_detection_time.load(Ordering::Relaxed),
             ),
-            remp_enabled: AtomicBool::new(self.remp_enabled.load(Ordering::Relaxed)),
+            remp_enabled: AtomicBool::new(self.remp_enabled.load(Ordering::Relaxed))
         }
     }
 }
 
 const QUERY_INFO_SCHEMA: &str = "?query=%7Binfo%7Bversion%20time%7D%7D";
 const QUERY_INFO_METRICS: &str = "?query=%7Binfo%7Bversion%20time%20latency%7D%7D";
-const QUERY_INFO_METRICS_REMP: &str =
-    "?query=%7Binfo%7Bversion%20time%20latency%20rempEnabled%7D%7D";
+const QUERY_INFO_METRICS_REMP: &str = "?query=%7Binfo%7Bversion%20time%20latency%20rempEnabled%7D%7D";
 
 const HTTP_PROTOCOL: &str = "http://";
 const HTTPS_PROTOCOL: &str = "https://";
 
 impl Endpoint {
-    pub fn http_headers(access_key: Option<&String>) -> Vec<(String, String)> {
-        let mut headers = vec![
+    pub fn http_headers() -> Vec<(String, String)> {
```

Comment:
This function must be declared and implemented as on the left side. You have to revert this function.